app/models/comment.rb

```diff
@@ -1,2 +1,4 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: :User, foreign_key: :user_id
+  belongs_to :post, class_name: :Post, foreign_key: :post_id
```

Comment:
- [ ] Foreign key isn't required at this point either. Simply do:
```ruby
belongs_to :post

Reply:
Done